cypress/e2e/ArticleViewing/article_content.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="cypress" />
+const { url, articlePage, textInPages } = require('../../support/selectors')
+
+describe('Make sure BBC.com loads properlly', () => {
+    beforeEach(() => {
+        cy.visit(url.articleChosenToTest);
+    });
+
+    it('Test Case 20. Verify article content', () => {
+        cy.title().should('include',textInPages.articleTitlePiece);
+        cy.get('time').should('be.visible');
+        cy.get(articlePage.videoPlayer).should('be.visible');
+        cy.get(articlePage.imgOfArthur).should('be.visible');
+    });
```

Comment:
bet cia testuoti statiskai viena article, jeigu istrins pavyzdziui sita straipsni tai testas taps visiskai bevertis?

Reply:
Taip. Kadangi article struktura skirtinga del to as nesugalvojau geresnio budo istestuot pati article apart kaip kad paziuret ar jis atsidaro ir ar ten yra raidziu. Del sios priezasties nutariau kad istestuot viena konkretu article yra efektyviau nei tiesiog paziuret ar jis atsidaro ir tiek.